### ISSUE TITLE
Person Bio Pages: Calendar should be in the right aside column

### ISSUE THREAD

**Issue by charlesbrandt** [City-of-Bloomington/drupal-theme-cob#36]
I recently configured Person/Bio pages to include a calendar field. Currently that calendar shows up below the details/body/about section. It would be better if it showed up under the picture in the right / aside column:

https://bloomington.in.gov/alpha/mayor

**Comment by inghamn** [comment]
We had talked about asides before, and declared that only the first aside would be to the right.  All the rest should show up below the about section.

Is this a confirmation that we have changed our minds, and now want all asides over to the right?

**Comment by schnippy** [comment]
@inghamn - aside from the above question, I'm still having issues with setting up the SSL / services connection to my account to allow pages with calendar fields to render. For the above page (/mayor), it fails with the following error:
```
The website encountered an unexpected error. Please try again later.

GuzzleHttp\Exception\RequestException: cURL error 60: SSL certificate problem: unable to get local issuer certificate (see http://curl.haxx.se/libcurl/c/libcurl-errors.html) in GuzzleHttp\Handler\CurlFactory::createRejection() (line 187 of /Users/gregschnippel/projects/bloomington/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php)
```

Would you mind if I try and add some fault tolerance to the output section of the calendar module (if I am right that this is where this is triggerd) to get around this error? I'm thinking of something that will test the connection and if it breaks, and $is_admin, then it will output a simple message in the space alerting that the calendar connection is not working otherwise leave it empty.

**Comment by inghamn** [comment]
I would be surprised if the unknown cert was one from Google.  There are other things on the screen that are webservice calls.  For instance, the contact info comes from an internal Directory web application.  It could be that module that's throwing the SSL error.

**Comment by inghamn** [comment]
I can take this one to avoid fiddling with your Calendar integration

**Comment by quarterback** [comment]
Just talked to Cliff about this. The calendar should probably be dredged up for situations where (like this one) the text is too long and so the calendar wraps too far. What we've come up with is for the page layout to stack in the aside in this order: 1) Aside text 2) Image 3) Calendar 4) Map (if the page has one).

**Comment by inghamn** [comment]
I've updated the alpha site with HTML / CSS to render all that stuff on the right, instead of any of it splitting up and going below the body field.

https://bloomington.in.gov/alpha/mayor

**Comment by charlesbrandt** [comment]
@schnippy I would be happy to have a bit more fault tolerance in the calls to integrated services. This would make it easier to test and debug outside of our local environment in cases where those other services are not available. Please let us know when those changes have been added.

**Comment by rbdietz** [comment]
This calendar (and all calendars on the site) should be updates to NOT say
"Google" calendar in the action button.





Rick Dietz

Director, Information & Technology Services Department
City of Bloomington, IN
dietzr@bloomington.in.gov
812.349.3485 Office
812.671.0776 Cell
bloomington.in.gov

**Comment by inghamn** [comment]
The calendar button was fixed in #37.  The change just had not been pushed to production.  It is there now.
